bases/shared/instagoric-server/server.js

```diff
@@ -668,6 +670,78 @@ faucetapp.get('/transaction-status/:txhash', (req, res) => {
   else res.status(400).send('invalid form');
 });
 
+publicapp.get('/install-bundle', async (req, res) => {
+  res.send(`
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>File Upload</title>
+</head>
+<body>
+
+    <h1>Upload a File</h1>
+
+    <form id="uploadForm" action="/install-bundle" method="POST" enctype="multipart/form-data">
```

Comment:
Why do we need to define `action` and `method` here if we are overriding the default browser behavior?

Reply:
updated